easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb

```diff
@@ -2093,9 +2103,6 @@ exts_list = [
     ('asnipe', '1.1.12', {
         'checksums': ['3a1f166f1c71b5877a2acca1384ec6c9b430b67af67ef26125f2abbb53c66206'],
     }),
-    ('liquidSVM', '1.2.4', {
```

Comment:
Why is `liquidSVM` dropped?

Reply:
Discussion from slack:

```
December 20 2019
Ed (Birmingham) 9:34 AM
Reequest for advice... the liquidSVN package in R 3.6.2 fosscuda is hard-wired for Intel:
* installing *source* package liquidSVM ...
** package liquidSVM successfully unpacked and MD5 sums checked
** using staged installation
Using target: generic
Using further args:
src/sources/shared/system_support/cuda_simple_vector_operations.cu
src/sources/shared/kernel/kernel_computation.cu
src/sources/svm/decision_function/decision_function_evaluation.cu
src/sources/svm/solver/validation_error_hinge_init.cu
src/sources/svm/solver/validation_error_computation.cu
Config in /scratch/edmondac-admin/eb-hUrqn1/Rtmp5M0zWq/R.INSTALLb787335904d1/liquidSVM
Config: using CXX_FLAG=-mtune=generic -msse2 -O3 $(CPP_CUDA_FLAGS)
Config: using ADDITIONAL_LIBS= $(CPP_CUDA_LINK_FLAGS)
Config: using ADDITIONAL_CPP_FLAGS= -DCOMPILE_SEPERATELY__CUDA
Config: using CUDA_PATH=/rds/bear-apps/devel/2019b/edmondac-eb-2/EL7/EL7-power9/software/CUDA/10.1.243-GCC-8.3.0
Config: using CUDA_OBJECTS= cuda_objects/cuda_simple_vector_operations.o cuda_objects/kernel_computation.o cuda_objects/decision_function_evaluation.o cuda_objects/validation_error_hinge_init.o cuda_objects/validation_error_computation.o
** libs
nvcc: creating cuda_simple_vector_operations.o
/rds/bear-apps/devel/2019b/edmondac-eb-2/EL7/EL7-power9/software/CUDA/10.1.243-GCC-8.3.0/bin/nvcc -I/rds/bear-apps/devel/2019b/edmondac-eb-2/EL7/EL7-power9/software/CUDA/10.1.243-GCC-8.3.0/include -arch sm_30 -L/rds/bear-apps/devel/2019b/edmondac-eb-2/EL7/EL7-power9/software/CUDA/10.1.243-GCC-8.3.0/lib64 -DCOMPILE_WITH_CUDA__ -U__SSE2__ -U__AVX__  -c sources/shared/system_support/cuda_simple_vector_operations.cu -o cuda_objects/cuda_simple_vector_operations.o \
  -I . -I../../../.. -I ../../../common  -DCOMPILE_SEPERATELY__CUDA -Xcompiler="-fPIC" -DCOMPILE_SEPERATELY__
/rds/bear-apps/devel/2019b/edmondac-eb-2/EL7/EL7-power9/software/GCCcore/8.3.0/include/c++/8.3.0/type_traits(335): error: identifier "__ieee128" is undefined
/rds/bear-apps/devel/2019b/edmondac-eb-2/EL7/EL7-power9/software/GCCcore/8.3.0/include/c++/8.3.0/bits/std_abs.h(101): error: identifier "__ieee128" is undefined
/rds/bear-apps/devel/2019b/edmondac-eb-2/EL7/EL7-power9/software/GCCcore/8.3.0/include/c++/8.3.0/bits/std_abs.h(102): error: identifier "__ieee128" is undefined
3 errors detected in the compilation of "/scratch/edmondac-admin/eb-hUrqn1/tmpxft_0000b7ce_00000000-6_cuda_simple_vector_operations.cpp1.ii".
make: *** [cuda_objects/cuda_simple_vector_operations.o] Error 1
ERROR: compilation failed for package liquidSVM
Option 1: Find a way to tell it not to use SSE2 and AVX on POWER chips
Option 2 (my preferred :wink: ) : Remove it from the R easyconfig entirely and have it separately if someone needs it. I don't yet know what other things depend on it...
What do you think?


ake_s 9:36 AM
Option 2 would be better


Ed (Birmingham) 9:39 AM
This makes the case for option 2 :wink: https://github.com/liquidSVM/liquidSVM/issues/5

rt3722rt3722
#5 Is this project actively maintained? looks like got removed from R packages.
<https://github.com/liquidSVM/liquidSVM|liquidSVM/liquidSVM>liquidSVM/liquidSVM | May 16th, 2019 | Added by GitHub
I'll remove it, pending anyone wanting to disagree :slightly_smiling_face:


boegel 10:02 AM
let's see if other stuff depends on it, to assess the impact?


Ed (Birmingham) 11:38 AM
OK - I'll do what's needed and include it in https://github.com/easybuilders/easybuild-easyconfigs/pull/9544 - and we can look at it in the new year :slightly_smiling_face:
```

Reply:
FTR: This was added not to long ago: https://github.com/easybuilders/easybuild-easyconfigs/pull/7597